Netbox_CSV_Read/CSV/format_dict.py

```diff
@@ -0,0 +1,69 @@
+from typing import Optional
+from Enums.dcim_device_id import DeviceInfoID
+from Enums.dcim_device_no_id import DeviceInfoNoID
+from Netbox_Api.netbox_connect import NetboxConnect
+from operator import attrgetter
+
+
+class FormatDict(NetboxConnect):
+
+    def __init__(self, url: str, token: str, dicts: list, api: Optional = None):
+        """
+        This method initialises the class with the following parameters.
+        Also, it allows dependency injection testing.
+        :param url: Netbox website URL.
+        :param token: Netbox authentication token.
+        :param dicts: A list of dictionaries to format.
+        """
+        if not api:
+            self.netbox = NetboxConnect(url, token).api_object()
+        else:
+            self.netbox = api
+        self.dicts = dicts
+        self.enums_id = DeviceInfoID
+        self.enums_no_id = DeviceInfoNoID
+
+    def iterate_dicts(self) -> list:
```

Comment:
This should be `List` and `Dict`, as `list != List` (it's one of those Python gotchas that was only fixed in later versions which we partially have access to)

Reply:
If the return typing should be uppercase List and Dict, does this apply to argument typing? like:
def func(values: List):
    pass
or
def func(values: list):
    pass